spritzle/tests/conftest.py

```diff
@@ -2,45 +2,29 @@
 from pathlib import Path
 import shutil
 
+from aiohttp import web
 import pytest
 
 from spritzle.core import Core
 from spritzle.config import Config
-from spritzle.tests.common import run_until_complete
+from spritzle.main import setup_app
 
-import asyncio
+pytest_plugins = 'aiohttp.pytest_plugin'
 
 
 @pytest.fixture(scope='function')
 def core():
-    # Create a new event loop for each core so that tests can't interfere with eachothers executor pools.
-    loop = asyncio.new_event_loop()
-    asyncio.set_event_loop(loop)
-    loop.set_debug(True)
-
     config = Config(in_memory=True, config_dir='/tmp')
     state_dir = Path(tempfile.mkdtemp(prefix='spritzle-test'))
     core = Core(config, state_dir)
-
-    @run_until_complete
-    async def core_start():
-        await core.start(
-            settings={
```

Comment:
It'd be nice to keep these settings for tests because we spawn a lot of libtorrent sessions and we sometimes try to make a lot of connections for dht/upnp/natpmp etc..

Reply:
Okay, I'll find a way to keep this.